src/test/lib/cachetasks_spec.js

```diff
@@ -21,7 +21,7 @@ var tRequest = testType(new Request(""));
 describe("CacheTasks", function() {
   describe("deleteCaches() method", () => {
     it("should delete all caches when no argument is given.", async(function*() {
-      var cacheNames = [CACHENAME, "foo", "bar", "baz", "bing", "skeleton_cache"];
+      var cacheNames = [CACHENAME, "foo", "bar", "baz", "bing", "skeleton_cache", "ads_cache"];
```

Comment:
This is going to keep biting us (it already bit me once, and seems it go you too), so maybe we should do: 

```JS
var cacheNames = [CACHENAME, "foo", "bar", "baz", "bing"].concat(yield caches.keys());
```
Then it will just take whatever already exists and trash everything.